package.json

```diff
@@ -42,12 +42,12 @@
     "karma-rollup-preprocessor": "^5.1.1",
     "mocha": "^5.0.0",
     "prettier": "^1.10.2",
-    "rollup": "^0.55.1",
+    "rollup": "^0.68.2",
```

Comment:
This is the requirement of rollup-plugin-typescript2.
https://github.com/ezolenko/rollup-plugin-typescript2#requirements